samples/OpenIdConnect.AzureAdSample/Readme.md

```diff
@@ -0,0 +1,20 @@
+# How to set up the sample locally
+
+## Set up [Azure Active Directory](https://azure.microsoft.com/en-us/documentation/services/active-directory/)
+
+1. Create your own Azure Active Directory (AD). Save the "tenent name".
+2. Add a new Application: in the Azure AD portal, select Application, and click Add in the drawer.
+3. Set the sign-on url to `http://localhost:5000`.
```

Comment:
The setting is in launchSettings.json, so VS will adopt it. However if you run the application from terminal using dotnet run, the port is 5000 instead.